Makefile

```diff
@@ -20,22 +20,45 @@ lint:
 	exit
 
 .PHONY: build
-build:
+build: generate-prod
 	go build -tags 'production' $(LDFLAGS) -o $(BINPATH)/dp-frontend-articles-controller
 
 .PHONY: debug
-debug:
+debug: generate-debug
 	go build -tags 'debug' $(LDFLAGS) -o $(BINPATH)/dp-frontend-articles-controller
 	HUMAN_LOG=1 DEBUG=1 $(BINPATH)/dp-frontend-articles-controller
 
 .PHONY: test
-test:
-	go test -race -cover ./...
+test: generate-prod
+	go test -race -cover -tags 'production' ./...
 
 .PHONY: convey
 convey:
 	goconvey ./...
 
 .PHONY: test-component
 test-component:
-	go test -cover -coverpkg=github.com/ONSdigital/dp-frontend-articles-controller/... -component
\ No newline at end of file
+	go test -cover -coverpkg=github.com/ONSdigital/dp-frontend-articles-controller/... -component
+
+.PHONY: generate-debug
+generate-debug: fetch-dp-renderer
+	# fetch the renderer library and build the dev version
+	cd assets; go run github.com/kevinburke/go-bindata/go-bindata -prefix $(CORE_ASSETS_PATH)/assets -debug -o data.go -pkg assets locales/... templates/... $(CORE_ASSETS_PATH)/assets/locales/... $(CORE_ASSETS_PATH)/assets/templates/...
+	{ echo "// +build debug\n"; cat assets/data.go; } > assets/debug.go.new
+	mv assets/debug.go.new assets/data.go
+
+.PHONY: generate-prod
+generate-prod: fetch-dp-renderer
+	# fetch the renderer library and build the prod version
+	cd assets; go run github.com/kevinburke/go-bindata/go-bindata -prefix $(CORE_ASSETS_PATH)/assets -o data.go -pkg assets locales/... templates/... $(CORE_ASSETS_PATH)/assets/locales/... $(CORE_ASSETS_PATH)/assets/templates/...
```

Comment:
Would it be better to add a version to go-bindata so the run happens in 'module aware' mode, i.e.
go run github.com/kevinburke/go-bindata/go-bindata@latest
and the module is downloaded and run without any reference to the go.mod file

Reply:
Thanks Patrick. I actually took this from the [dp-renderer migration guide](https://github.com/ONSdigital/dp-renderer/blob/main/MIGRATION.md#updating-the-makefile) so I'd suggest leaving it